src/main/java/org/jabref/gui/actions/ManageKeywordsAction.java

```diff
@@ -156,7 +187,28 @@ public void keyReleased(KeyEvent arg0) {
             @Override
             public void keyPressed(KeyEvent arg0) {
                 if (arg0.getKeyCode() == KeyEvent.VK_DELETE) {
-                    removeActionListenter.actionPerformed(null);
+                    removeActionListener.actionPerformed(null);
+                }
+            }
+        });
+
+        // enable a user to press Enter to get a keyword added
+        keyword.addKeyListener(new KeyListener() {
+
```

Comment:
Use keyAdapter instead of key listener, so you don't need to have the empty method.